versions.json

```diff
@@ -279,8 +279,8 @@
     },
     "libs3": {
         "enable_sha": true,
-        "commitish": "e4197a5e02f03059c48ecac9ed240c66d55e15f6",
-        "version_string": "e4197a5e",
+        "commitish": "c0e278d280cde08fb588d82dcb125198852f2bc9",
+        "version_string": "c0e278d28",
```

Comment:
Why does the version need to be bumped?

Reply:
This commit (https://github.com/irods/libs3/commit/c0e278d280cde08fb588d82dcb125198852f2bc9) adds the `xAmzStorageClass` member to the `S3PutProperties` struct. `S3PutProperties::xAmzStorageClass` is first used here in the S3 resource plugin: https://github.com/irods/irods_resource_plugin_s3/commit/3f10641132bc761e4fc7112c7ba4ba881ee1588c#diff-55f46ae97682d04ee9ce78c3905019815797351886935b8e3396407dace53cb7R1710

Reply:
I see. Proceed.